save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/organizations/OrganizationTestsMenu.kt

```diff
@@ -204,6 +228,20 @@ private fun prepareTestSuitesSourcesTable(
                 }
             }
         }
+        column(id = "fetch", header = "Fetch new version", { this }) { cellProps ->
+            Fragment.create {
+                td {
+                    button {
+                        type = ButtonType.button
+                        className = ClassName("btn btn-sm btn-primary")
+                        onClick = {
+                            fetchHandler(cellProps.value)
+                        }
+                        +"fetch"
```

Comment:
Fetch will start in the backgroud, right? Maybe we should display some kind of confirmation that request is accepted and the version will be updated later?